src/flowcean/environments/rosbag.py

```diff
@@ -0,0 +1,84 @@
+from os import listdir
+from pathlib import Path
+from typing import Self, override
+
+import polars as pl
+from rosbags.dataframe import get_dataframe
+from rosbags.highlevel import AnyReader
+from rosbags.typesys import Stores, get_types_from_msg, get_typestore
+
+from flowcean.core.environment import OfflineEnvironment
+
+
+def guess_msgtype(path: Path) -> str:
+    """Guess message type name from path."""
+    name = path.relative_to(path.parents[2]).with_suffix("")
+    if "msg" not in name.parts:
+        name = name.parent / "msg" / name.name
+    return str(name)
```

Comment:
It's not clear to me what this function does. Can you either add inline comments to explain your thinking or add an example of the functions input/output in the docstring?

Reply:
Added example usage with input/output to docstring